app/views/report/_report_info.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render :partial => "layouts/flash_msg"%>
+<%= report_info_summary(@sb, @miq_report, @schedules, @widget_nodes)%>
```

Comment:
I'm not sure this matters, but at least stylistically, I'd expect a space before the closing `%>`

```suggestion
<%= render :partial => "layouts/flash_msg" %>
<%= report_info_summary(@sb, @miq_report, @schedules, @widget_nodes) %>
```